processing/src/main/java/io/druid/guice/ExtensionsConfig.java

```diff
@@ -41,6 +41,10 @@
   @JsonProperty
   private String hadoopContainerDruidClasspath = null;
 
+  //Only applicable when hadoopContainerDruidClasspath is explicitly specified.
+  @JsonProperty
+  private boolean addExtensionsToHadoopContainer = false;
```

Comment:
Actually, when the user has not set the hadoopContainerDruidClasspath, we always add extensions to hadoop classpath. Can we call it out in the docs that if  druid.extensions.hadoopContainerDruidClasspath is null, extensions will always be added to the classpath.